src/java.base/share/classes/java/io/Console.java

```diff
@@ -390,13 +391,31 @@ public void flush() {
         pw.flush();
     }
 
+
+    /**
+     * Returns the {@link java.nio.charset.Charset Charset} object used in
+     * this {@code Console}.
```

Comment:
The Console is a singleton and the existing methods use "the console" so I think we should do the same here.

We'll need to add to the description of the System.{in,out,err} fields, I don't mind if we do it as part of this PR or another issue.

Reply:
Javadoc updated. Instead of modifying System.out/err (System.in is not affected, as it does not convert b2c), I modified PrintStream javadoc.

Reply:
Reverted the changes to PrintStream, as it defaults to Charset.defaultCharset(). Added descriptions to System.out/err instead (and modified the impl).